src/twisted/conch/ssh/transport.py

```diff
@@ -24,6 +24,13 @@
 from cryptography.hazmat.primitives import serialization
 from cryptography.hazmat.primitives.asymmetric import dh, ec, x25519
 from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+
+try:
+    from cryptography.hazmat.primitives.decrepit.ciphers import TripleDES
```

Comment:
Shouldn't this be:

```suggestion
    from cryptography.hazmat.decrepit.ciphers import TripleDES
```

At least, that matches https://cryptography.io/en/latest/hazmat/decrepit/ciphers/#cryptography.hazmat.decrepit.ciphers.TripleDES and I guess it'd fix mypy.

Reply:
Love that I messed up my own import. 😅